### PR TITLE
add page

### PR DESCRIPTION
**Pull request by karleywalker**

### CODE REVIEW

**Comment by karleywalker** [comment]
@githubteacher

File: _pages/karleywalker.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: karleywalker
+pageURL: https://karleywalker.github.io
```
Comment (hectorsector) [inline _pages/karleywalker.yaml]: @karleywalker remove the `https://` and that should get that test to pass

File: _pages/karleywalker.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+timestamp: 2017-01-25
+---
+
```
Comment (hectorsector) [inline _pages/karleywalker.yaml]: @karleywalker you should only need one new-line character

**Comment by brianamarie** [comment]
Hi @karleywalker! I've made some changes. Since it's been sitting for a while and the PR is ready, I'm going to go ahead and merge. Nice work!

**Pull request merged**